src/Elastic.Apm.NLog/Elastic.Apm.NLog.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
https://github.com/elastic/ecs-dotnet/blob/master/src/Elastic.CommonSchema.Serilog/Elastic.CommonSchema.Serilog.csproj#L3

Needs to import props similar to linked projecf so that versions and other metadata are set.